CQUI.modinfo

```diff
@@ -836,13 +836,13 @@
                 <Context>InGame</Context>
             </Properties>
             <Items>
-                <File>Integrations/ML/Lenses/ModLens_Builder.lua</File>
-                <File>Integrations/ML/Lenses/ModLens_Scout.lua</File>
-                <File>Integrations/ML/Lenses/ModLens_Archaeologist.lua</File>
-                <File>Integrations/ML/Lenses/ModLens_Barbarian.lua</File>
-                <File>Integrations/ML/Lenses/ModLens_Naturalist.lua</File>
-                <File>Integrations/ML/Lenses/ModLens_Wonder.lua</File>
-                <File>Integrations/ML/Lenses/ModLens_CitizenManagement.lua</File>
+                <File>Integrations/ML/Lenses/Archaeologist/ModLens_Archaeologist.lua</File>
```

Comment:
Why is each lens in a separate folder? In my ML install from Steam all lenses are directly in the Lens folder, no sub-folders.